workspaces/liferay-partner-workspace/quickstart/liferay.env

```diff
@@ -10,6 +10,7 @@ LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_DRIVER_UPPERCASEC_LASS_UPPERCASEN_AME=com.mys
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_PASSWORD=
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_URL=jdbc:mysql://database/lportal?useUnicode=true&characterEncoding=UTF-8
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_USERNAME=root
+LIFERAY_JPDA_ENABLED=true
```

Comment:
What does this flag do for us? or what problem does it solve?

Reply:
I've found it in this [slack thread](https://liferay.slack.com/archives/C01CSLH1UFJ/p1661350164526959).
Well, it worked without this other flag he mentioned `JPDA_ADDRESS=0.0.0.0:8000`, I can try with just the 8000 port exposed to see if this is really a requirement.